vagrant/etc/ImageMagick-6/policy.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policymap [
+<!ELEMENT policymap (policy)+>
+<!ELEMENT policy (#PCDATA)>
+<!ATTLIST policy domain (delegate|coder|filter|path|resource) #IMPLIED>
+<!ATTLIST policy name CDATA #IMPLIED>
+<!ATTLIST policy rights CDATA #IMPLIED>
+<!ATTLIST policy pattern CDATA #IMPLIED>
+<!ATTLIST policy value CDATA #IMPLIED>
+]>
+<policymap>
+  <policy domain="coder" rights="read|write" pattern="EPHEMERAL" />
+  <policy domain="coder" rights="read|write" pattern="URL" />
+  <policy domain="coder" rights="read|write" pattern="HTTPS" />
```

Comment:
I don't know what most of these mean, but granting read and write permissions to network resources doesn't sound great to me:

> If you utilize ImageMagick from a public website, you may want to increase security by preventing usage of the MVG or HTTPS coders.

It's wild that this is even a capability ImageMagick has built!